packages/x-charts-pro/src/ChartsToolbarPro/ChartsToolbarPro.tsx

```diff
@@ -28,22 +58,118 @@ export function ChartsToolbarPro(props: ChartsToolbarProProps) {
     children.push(
       <Tooltip key="zoom-in" {...slotProps.baseTooltip} title={localeText.zoomIn}>
         <ChartsToolbarZoomInTrigger render={<ToolbarButton />}>
-          <ZoomInIcon {...slotProps.zoomInIcon} />
+          <ZoomInIcon fontSize="small" {...slotProps.zoomInIcon} />
         </ChartsToolbarZoomInTrigger>
       </Tooltip>,
     );
     children.push(
       <Tooltip key="zoom-out" {...slotProps.baseTooltip} title={localeText.zoomOut}>
         <ChartsToolbarZoomOutTrigger render={<ToolbarButton />}>
-          <ZoomOutIcon {...slotProps.zoomOutIcon} />
+          <ZoomOutIcon fontSize="small" {...slotProps.zoomOutIcon} />
         </ChartsToolbarZoomOutTrigger>
       </Tooltip>,
     );
   }
 
+  if (showExportMenu) {
+    const Tooltip = slots.baseTooltip;
+    const MenuList = slots.baseMenuList;
+    const MenuItem = slots.baseMenuItem;
+    const ExportIcon = slots.exportIcon;
+
+    const closeExportMenu = () => setExportMenuOpen(false);
+
+    const handleListKeyDown = (event: React.KeyboardEvent) => {
+      if (event.key === 'Tab') {
+        event.preventDefault();
+      }
+
+      if (isHideMenuKey(event.key)) {
+        closeExportMenu();
+      }
+    };
+
+    if (children.length > 0) {
+      children.push(<ChartsToolbarDivider key="divider" />);
+    }
+
+    children.push(
+      <React.Fragment key="export-menu">
+        <Tooltip title={localeText.toolbarExport}>
+          <ToolbarButton
+            ref={exportMenuTriggerRef}
+            id={exportMenuTriggerId}
+            aria-controls={exportMenuId}
+            aria-haspopup="true"
+            aria-expanded={exportMenuOpen ? 'true' : undefined}
+            onClick={() => setExportMenuOpen(!exportMenuOpen)}
+          >
+            <ExportIcon fontSize="small" />
+          </ToolbarButton>
+        </Tooltip>
+
+        <ChartsMenu
+          target={exportMenuTriggerRef.current}
+          open={exportMenuOpen}
+          onClose={closeExportMenu}
+          position="bottom-end"
```

Comment:
Nitpick but `bottom-end` feels weird on a centered toolbar. I would expect a `bottom-start`

Like in G-docs for example.

![image](https://github.com/user-attachments/assets/02d6cb7d-7aac-443f-aa5c-40d3180a4870)

I suspect the data grid went that way because the button is on the far right of the grid which can easily be a full screen feature.

Reply:
About toolbar sizing, I've open a discussion to reduce size https://www.figma.com/design/Sl9YzrZlA7RlowwEfYLWDz?node-id=130-1026#1290522663

Reply:
> Nitpick but bottom-end feels weird on a centered toolbar. I would expect a bottom-start

Looks good to me. @KenanYusuf @noraleonte do you see any drawbacks? If so, let me know, and I'll revert this change.

Reply:
I like `bottom-end` in this case, but happy to go with the majority.